Cargo.toml

```diff
@@ -17,23 +17,24 @@ include = ["/src", "LICENSE", "README.md", "CHANGELOG.md"]
 thiserror = "1.0"
 log = "0.4"
 imgui = "^0.12"
-ash = { version = ">=0.34, <=0.37", default-features = false, features = ["debug"] }
+ash = { version = "0.38", default-features = false, features = ["debug"] }
 ultraviolet = "0.9"
 
-gpu-allocator = { version = "0.26", default-features = false, features = ["vulkan"], optional = true }
+#gpu-allocator = { version = "0.26", default-features = false, features = ["vulkan"], optional = true }
+gpu-allocator = { git = "https://github.com/Traverse-Research/gpu-allocator.git", branch = "ash-0.38", default-features = false, features = ["vulkan"], optional = true }
 
-vk-mem = { version = "0.3", optional = true }
+vk-mem = { version = "0.4", optional = true }
 
 [features]
 dynamic-rendering = []
 
 [dev-dependencies]
 simple_logger = "5.0"
-winit = { version = "0.29", default-features = false, features = ["rwh_05", "x11", "wayland", "wayland-dlopen", "wayland-csd-adwaita"]}
+winit = { version = "0.29", default-features = false, features = ["rwh_06", "x11", "wayland", "wayland-dlopen", "wayland-csd-adwaita"]}
```

Comment:
You can go back to `winit = "0.29` now

Reply:
Done but I am waiting for https://github.com/adrien-ben/imgui-rs-vulkan-renderer/pull/44 to be merged as I have already rebased this branch on it.